modules/cloudwatch-log-based-metrics/tests/basic/0-setup.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_version = "~> 1.3"
+
+  required_providers {
+    test = {
+      source = "terraform.io/builtin/test"
+    }
+
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.33"
+    }
+  }
+}
+
+provider "aws" {}
+
+# prepare test log group
+resource "aws_cloudwatch_log_group" "log_group_test" {
```

Comment:
Is there a way to skip those checks for tests folder? or we should implement those properly?